tests/other/metadce/mem_no_main_O3_STANDALONE_WASM.funcs

```diff
@@ -1,4 +1,5 @@
-$__wasm_call_ctors
+$__main_void
+$_start
```

Comment:
seems like before they were inlined into one function? or main was a duplicate of `wasm_call_ctors` (empty). So might be worth finding out why that changed.